chatbot/python/chatbot.py

```diff
@@ -309,6 +310,8 @@ def on_login(cookie_file_name, params):
 
     if 'user' in params:
         botUID = params['user'].decode("ascii")
+        if botUID[0] == '"':
```

Comment:
Why is this needed?

Reply:
`params['user'].decode("ascii")` returns the string containing quotation marks in the front and end, like '"usrxxxxx"'. I am not sure it will always happen in any environments.

Reply:
It's always quoted because values `params` dictionary are bytes. So, just always unquote it:

```python
# Values params are bytes, so a string is always quoted. Unquote it.
botUID = params['user'].decode("ascii")[1:-1]
```

Reply:
Thanks, got it and fixed.